uniro_frontend/src/index.css

```
@@ -136,3 +136,19 @@
 @utility translate-waypoint {
 	transform: translateY(+4px);
 }
+
+@keyframes markerAppear {
+	0% {
+		opacity: 0;
+		transform: scale(0);
+	}
+	100% {
+		opacity: 1;
+		transform: scale(1);
+	}
+}
+
+.marker-appear {
+	animation: markerAppear 0.3s ease-out forwards;
+	transform-origin: bottom center;
+}
```

figma 차이가 있어서 시안대로 적용하는게 좋을 것 같습니다.
```suggestion
.marker-appear {
	animation: markerAppear 0.2s ease-in-out forwards;
	transform-origin: bottom center;
}
```